libs/cardano-ledger-binary/test-lib/Test/Cardano/Ledger/Binary/Twiddle.hs

```diff
@@ -65,7 +87,7 @@ instance Twiddle a => Twiddle [a] where
     l' <- traverse twiddle l
     pure $ f l'
 
-instance (Twiddle k, Twiddle v) => Twiddle (Map k v) where
+instance (Twiddle k, Twiddle v, Ord k) => Twiddle (Map k v) where
```

Comment:
Just curious, what change requires `Ord`?

Reply:
I couldn't pin down exactly which function needs it, but I guess generally `Map`s implicitly assume the key to be an `Ord` instance.

Reply:
It is the `FromCBOR` super class constraint on the `Twiddle` class. Because `FromCBOR (Map k v)` requires `Ord k`, now the `Twiddle (Map k v)` requires `Ord k`